assets/js/copyable.js

```diff
@@ -0,0 +1,33 @@
+async function copyText(e) {
+    const btn = e.currentTarget;
+    const icon = btn.firstChild;
+    const textNode = btn.previousElementSibling;
+    try {
+        await navigator.clipboard.writeText(textNode.textContent);
+        icon.classList.remove('fa-copy');
+        icon.classList.add('fa-check');
+      } catch (err) {
```

Comment:
Looks like I messed up the indentation on this line.

```suggestion
    } catch (err) {
```